content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md

```diff
@@ -19,7 +19,7 @@ aliases:
 # 0 -- hidden
 # 1 -- visible
 # 2 -- highlighted
-weight: 2
```

Comment:
Just curious, is there a limit on the weight? Going by the comment I only see `0, 1, 2`.

Reply:
I played around with these values to get the ordering right as requested by Ammie.
Digital.gov convention would be to use `0, 1, 2` but any integer is allowed.

Reply:
Okay, thanks Nick. I don't think we should stray too far from this guidance. For example, maybe not go higher than a certain weight. Just something to consider for the future.